### PR TITLE
Code cleanups

### PR DESCRIPTION
**Pull request by nhosoi**
- Removing unnecessary skip_ansible_lint tags.
- Replacing include with include_tasks.

### CODE REVIEW

**Comment by nhosoi** [comment]
[citest bad]

**Comment by nhosoi** [comment]
[citest bad]

**Comment by nhosoi** [comment]
[citest bad]

**Comment by nhosoi** [comment]
[citest pending]

**Comment by nhosoi** [comment]
[citest bad]

**Comment by nhosoi** [comment]
@richm, I'm having difficulty troubleshooting the CI test failures...
Although tests under linux-system-roles-test passed, all of them under linux-system-roles-test-staging failed.
Not as before, the test results are not helpful to understand the problem. (well, I don't understand why this PR causes these many failures, but putting it aside... :)
For instance, in this [staging CentOS-7 result](https://fedorapeople.org/groups/linuxsystemroles/logs/linux-system-roles-logging-pull-linux-system-roles_logging-152-558fa7b-centos-7-20200708-214417/artifacts/test.log.htm), this is the last part of [ansible.log](https://fedorapeople.org/groups/linuxsystemroles/logs/linux-system-roles-logging-pull-linux-system-roles_logging-152-558fa7b-centos-7-20200708-214417/artifacts/ansible.log).
```
TASK [Run logger to generate a test log message] *******************************
task path: /tmp/tmpf751ejqp/tests/tests_basics_files_forwards.yml:104
```
It looks it's just cut there. Do you have any idea what is wrong? And how can I get the full result?

**Comment by nhosoi** [comment]
nbde_client is done: https://github.com/linux-system-roles/nbde_client/pull/10/

**Comment by richm** [comment]
> It looks it's just cut there. Do you have any idea what is wrong? And how can I get the full result?

Sorry - I was altering the staging environment in order to debug the certificate tests.  Just ignore those for now.

**Comment by nhosoi** [comment]
> > It looks it's just cut there. Do you have any idea what is wrong? And how can I get the full result?
> 
> Sorry - I was altering the staging environment in order to debug the certificate tests. Just ignore those for now.

*relieved* :) thanks!

**Comment by nhosoi** [comment]
[citest bad]

**Comment by nhosoi** [comment]
[citest bad]

File: tests/tests_basics_forwards_cert.yml [hunk 1]
```diff
@@ -80,7 +80,7 @@
       include_role:
         name: linux-system-roles.logging
 
-    - include: set_rsyslog_variables.yml
```
Comment (pcahyna) [inline tests/tests_basics_forwards_cert.yml:83]: Maybe use `import_tasks` while you are at it, if you don't mind its limitations? I heard some arguments in favor of imports vs includes. Notably includes have some performance problems, and imports also allow more static analysis (linting) iirc.

Comment (nhosoi) [reply tests/tests_basics_forwards_cert.yml:83]: @pcahyna, is it ok for me to replace all the include_tasks in the tests playbooks? There are more than a dozen...

Comment (richm) [reply tests/tests_basics_forwards_cert.yml:83]: > @pcahyna, is it ok for me to replace all the include_tasks in the tests playbooks? There are more than a dozen...

Not sure - there are slightly different behaviors of import vs. include - I would prefer to keep the `include` behavior for now in this PR.  If we find that using `import` is a significant advantage, we should plan to change all of the roles to use `import` where possible.

Comment (nhosoi) [reply tests/tests_basics_forwards_cert.yml:83]: I think there's no much impact in the replacement. I'm reverting [this second commit](https://github.com/linux-system-roles/logging/pull/152/commits/1d126798c3987f28389ab981cdb514e65dfba2e5) for now.

**Comment by pcahyna** [comment]
Why don't you just reset the branch to the first commit?

**Comment by nhosoi** [comment]
> Why don't you just reset the branch to the first commit?

I thought it was the way to squash into one or some commits just before the merge...
But of course, I can do it now. :)

**Comment by nhosoi** [comment]
Thanks, @pcahyna. Merging...

**Pull request merged**